mephisto/scripts/local_db/review_feedback_for_task.py

```diff
@@ -0,0 +1,227 @@
+from mephisto.tools.data_browser import DataBrowser as MephistoDataBrowser
+
+yes_no_responses = set(["yes", "y", "YES", "Yes", "no", "n", "NO", "No"])
+yes_response = set(["yes", "y", "YES", "Yes"])
+no_response = set(["no", "n", "NO", "No"])
+
+
+def set_feedback_as_reviewed(feedback: List, id: str, unit: Unit) -> None:
+    """Sets the reviewed property of a feedback to true"""
+    assigned_agent = unit.get_assigned_agent()
+    feedback_ids = [feedback_obj["id"] for feedback_obj in feedback]
+    index_to_modify = get_index_of_value(feedback_ids, id)
+    if assigned_agent is not None:
+        feedback[index_to_modify]["reviewed"] = True
+        assigned_agent.state.update_metadata(
+            property_name="feedback", property_value=feedback
+        )
+
+
+def print_out_feedback_elements(
+    filtered_feedback_list: List[Dict[str, Any]],
+    is_unreviewed: bool,
+    unit: Unit,
+    feedback: List[Dict[str, Any]],
+) -> None:
+    """Prints out the desired feedback object properties from a filtered list"""
+
+    for feedback_obj in filtered_feedback_list:
+        print("Feedback Id: " + feedback_obj["id"])
+        print("Feedback Question: " + feedback_obj["question"])
+        print("Feedback Text: " + feedback_obj["text"])
+        print("Feedback Toxicity: " + feedback_obj["toxicity"])
+        print("")
+
+        """
+        The option to mark feedback as reviewed should be
+        presented when viewing unreviewed feedback.
+        """
+
+        if is_unreviewed == True:
+            mark_feedback_as_reviewed = input(
+                "\nDo you want to mark this feedback as reviewed? yes(y)/no(n): "
+            )
+            while mark_feedback_as_reviewed not in yes_no_responses:
+                print("That response is not valid\n")
+                mark_feedback_as_reviewed = input(
+                    "\nDo you want to mark this feedback as reviewed? yes(y)/no(n): "
+                )
+            if mark_feedback_as_reviewed in yes_response:
+                set_feedback_as_reviewed(feedback, feedback_obj["id"], unit)
+                print("\nMarked the feedback as reviewed!")
+            elif mark_feedback_as_reviewed in no_response:
+                print("\nDid not mark the feedback as reviewed!")
+            print("\n")
+
+
+def main():
+    db = LocalMephistoDB()
+    mephisto_data_browser = MephistoDataBrowser(db)
+    task_names = mephisto_data_browser.get_task_name_list()
+    print("\nTask Names:")
+    for task_name in task_names:
+        print(task_name)
+
+    # Getting the task name
+    task_name = input(
+        "\nEnter the name of the task that you want to review the feedback of: \n"
+    )
+    print("")
+    while task_name not in task_names:
+        print("That task name is not valid\n")
+        task_name = input(
+            "Enter the name of the task that you want to review the feedback of: \n"
+        )
+        print("")
```

Comment:
As an aside, this style of task name selection is a pretty common utility, and would likely be a good candidate to abstract into the `mephisto.tools.scripts` collection. Can do after #816